### PR TITLE
Adding pywaterml 

### PR DESCRIPTION
**Pull request by romer8**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by romer8** [comment]
I am willing to be listed there

File: LICENSE.txt [hunk 1]
```diff
@@ -1,27 +1,14 @@
+BSD 3-clause license
```
Comment (dopplershift) [inline LICENSE.txt:1]: This is the license file for the staged-recipes repository and should not be changed by this PR.

File: recipes/pywaterml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  sha256: 636118f005a451e9ef64dbdeb47a7e518a8f8684bb1fcd07d9607fe9473ff618
+
+build:
+
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```
Comment (dopplershift) [inline recipes/pywaterml/meta.yaml:19]: The `host` section really is just what's needed to run `setup.py`, not the full package. It looks like this should only need to be:
```yaml
- python >=3.6
- pip
- setuptools
```

**Review by dopplershift — CHANGES_REQUESTED** [review]

File: LICENSE.txt (hunk 1 above)
Comment (romer8) [reply LICENSE.txt:1]: I change back to the original LICENSE

File: recipes/pywaterml/meta.yaml (hunk 2 above)
Comment (romer8) [reply recipes/pywaterml/meta.yaml:19]: I changed it to only used the suggested dependecies in the host

**Review by dopplershift — APPROVED** [review]

**Pull request merged**